eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/implementation/RequestResponseChannel.java

```diff
@@ -137,7 +138,7 @@ private void start() {
     private void send(final Message message) {
         sendLink.delivery(UUID.randomUUID().toString().replace("-", "").getBytes(UTF_8));
 
-        final int payloadSize = AmqpUtil.getDataSerializedSize(message) + 512; // need buffer for headers
+        final int payloadSize = EventDataUtil.getDataSerializedSize(message) + 512; // need buffer for headers
```

Comment:
Should we consider putting that 512 into the `AmqpConstants` to keep all the protocol-specifics in one place?